src/ir_Bosch.h

```diff
@@ -0,0 +1,166 @@
+// Copyright 2022 David Conran
```

Comment:
Your name here, not mine. It's your work & effort! ;-)

Reply:
changed